automation/UNIX/run-cmake.sh

```diff
@@ -6,16 +6,12 @@ set -ue
 set -o pipefail
 set -x
 
-if [ "${TEST_TARGET_ARCH}" == "X86" ]; then
-  CMAKE_ADDITIONAL_OPTIONS="-DLLVM_TARGET_ARCH=X86"
+if [[ ("$LNT" != "" || "$BUILD_PACKAGE" == "Yes") && ("$BUILDCONFIGURATION" == "Release" || "$BUILDCONFIGURATION" == "ReleaseWithDebInfo") ]]; then
```

Comment:
When building a package, you also want to set -DLLVM_INSTALL_TOOLCHAIN_ONLY=ON.  Otherwise the install package will be much larger than necessary.